### ISSUE TITLE
Feature Request: Energymonitoring

### ISSUE THREAD

**Issue by betonishard** [1technophile/OpenMQTTGateway#208]
Hi all,
Great project (again). I was wondering If there is a need for adding an LM393 tot OpenMQTT.. I would use it like in this project: https://www.mysensors.org/build/pulse_power

I have a gateway set up in my electrical room. Would it be possible to add this, or manually replace one.?
Regards,

**Comment by betonishard** [comment]
Maybe to make it more generic. A pulse counter

**Comment by 1technophile** [comment]
Yes if someone has this kind of sensor don't hesitate [to develop ](https://github.com/1technophile/OpenMQTTGateway/blob/master/CONTRIBUTING.md)a new ZsensorLM393 and a config_LM393 so as to intégrate it into OMG.

**Comment by betonishard** [comment]
That would be great, if only there was someone,...
